### PR TITLE
Automated release of 25.2.0a1 upstream release 25.0 1665521856

### PR DESCRIPTION
**Pull request by yliaog**
#### What type of PR is this?
/kind bug

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ supported versions of Kubernetes clusters.
 - [client 22.y.z](https://pypi.org/project/kubernetes/22.6.0/): Kubernetes 1.21 or below (+-), Kubernetes 1.22 (✓), Kubernetes 1.23 or above (+-)
 - [client 23.y.z](https://pypi.org/project/kubernetes/23.6.0/): Kubernetes 1.22 or below (+-), Kubernetes 1.23 (✓), Kubernetes 1.24 or above (+-)
 - [client 24.y.z](https://pypi.org/project/kubernetes/24.2.0/): Kubernetes 1.23 or below (+-), Kubernetes 1.24 (✓), Kubernetes 1.25 or above (+-)
+- [client 25.y.z](https://pypi.org/project/kubernetes/25.2.0/): Kubernetes 1.24 or below (+-), Kubernetes 1.25 (✓), Kubernetes 1.26 or above (+-)
```
Comment (roycaihw) [inline README.md:97]: 25.2.0a1? LGTM otherwise

**Review by roycaihw — COMMENTED** [review]
/lgtm
/approve

File: README.md (hunk 1 above)
Comment (yliaog) [reply README.md]: fixed

**Pull request merged**